waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/conf/WaggleDanceConfiguration.java

```diff
@@ -118,4 +119,12 @@ public void setThriftServerRequestTimeoutUnit(TimeUnit thriftServerRequestTimeou
     this.thriftServerRequestTimeoutUnit = thriftServerRequestTimeoutUnit;
   }
 
+  public Long getStatusPollingDelay() {
+    return statusPollingDelay;
+  }
+
+  public void setStatusPollingDelay(Long statusPollingDelay) {
+    this.statusPollingDelay = statusPollingDelay;
+  }
+
```

Comment:
To be consistent with `disconnect-connection-delay` shouldn't we also have a TimeUnit for this rather than forcing ms?

Reply:
Then we can't use the annotation anymore. The spring schedule doesn't have a TimeUnit

Reply:
Bugger, that's a pity as we now have an inconsistent way of configuring "time related things". I guess the alternative would be that we do it the consistent way here and then convert it into ms before we call spring? If we only allow it to be configured here and then set it later that would be nice but I see your point on the annotation, you're saying it can't be given a value other than via a property? Would we have to programmatically do some stuff in the constructor or somewhere else instead?

Reply:
We could not sue the annotations and make a schedule bean/class/thing programatically (I think). Then we can control it better. I felt that was overkill but I don't mind doing it.

Reply:
Yeah, I think since a lot of our "contract" is actually what end users see in the yaml file we should make that as simple and consistent as possible. As an end user I think I'd find it confusing if some times are specified with a unit and some not, I'd prefer either one or the other wouldn't you? If you agree then I think we should do the extra work in the code to make this better for them.